lib/system/zephyr/assert.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2018, Nordic Semiconductor ASA and Contributors.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of Xilinx nor the names of its contributors may be used
```

Comment:
The should the license text be "Xilinx" or Nordic?

Reply:
I think it better to change, otherwise, "Xilinx" should be in the copyright text.